Locales/tst/TerminalCategoryTopos.tst

```diff
@@ -0,0 +1,222 @@
+###############################################################
+# Test the (co)cartesian (co)closed correspondences between
+# the terminal category with multiple objects and its opposite.
+###############################################################
+
+gap> START_TEST("TerminalCategoryTopos.tst");
+
+gap> LoadPackage("Locales", false);;
+
+gap> T := TerminalCategoryWithMultipleObjects( : FinalizeCategory := false );;
+
+gap> AddDerivationToCAP( UniqueMorphism, function( cat, A, B ) return CreateCapCategoryMorphismWithAttributes( cat, A, B ); end : CategoryFilter := IsThinCategory );;
```

Comment:
This is dangerous because it affects all tests afterwards. Can't you simply call `AddUniqueMorphism` manually?

Reply:
I tried AddUniqueMorphism in the first version of this PR, but did not cover the bug fix, as can be seen in codecov. Now it is failing anyway.